README.md

```diff
@@ -76,7 +76,9 @@ The OpenGApps AOSP based build system is placed in `vendor/google/build` and inc
 in `build/core/main.mk`:
 ```
 # Include the google-specific config
+ifneq ($(GAPPS_VARIANT),)
```

Comment:
Ok...I just hope Opengapps to better support Other rom... such as:Cyanogenmod,Thanks